RetailSuiteAccounting/Operation/06_DeleteBookings.md

```diff
@@ -0,0 +1,72 @@
+- A valid fiscal year is created, see [Create a fiscal year](../Integration/04_ManageFiscalYear.md#create-a-fiscal-year).
+- A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).
+- A booking has been created, see [Create a manual booking](./04_CreateManualBooking.md).
+
+#### Procedure
+
+*Accounting > Select fiscal year > Book > Tab BOOKINGS*
+
+![Delete booking](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/DeleteBooking.png "[Delete booking]")
+
+1. Select the booking to be deleted from the list in the *BOOKINGS* tab.
+
+2. The existing booking details are displayed in the input line fields.
+
+3. Click the [DELETE] button.  
+A confirmation window is displayed.
+
+  > [Warning] Be aware that the existing booking will be deleted permanently.
+
+4. Click the [Ok] button to confirm the action.  
+A message above the input line confirms that the booking has been deleted. The deleted booking will no longer be displayed in the bookings list.
+
+  ![Booking deleted](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/BookingDeleted.png "[Booking deleted]")
+
+
+## Delete multiple bookings
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
+- A valid fiscal year is created, see [Create a fiscal year](../Integration/04_ManageFiscalYear.md#create-a-fiscal-year).
+- A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).
+- Multiple bookings have been created, see [Create a manual booking](./04_CreateManualBooking.md).
+
+#### Procedure
+
+*Accounting > Select fiscal year > Book > Tab BOOKINGS > Context menu EDIT > Menu entry Delete*
+
+![Delete multiple bookings](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/DeleteMultipleBookings.png "[Delete multiple bookings]")
+
+1. Click the *Delete* menu entry in the *EDIT* context menu.   
+A window opens for you to select the journal and the sequential numbers of the bookings to be deleted.
+
+  ![Select multiple bookings](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/DeleteMultipleBookings01.png "[Select multiple bookings]")
+
+2. Select the journal where the bookings to be deleted are recorded in the *Journal* drop-down list.
+
+3. Enter the number range of the bookings to be deleted in the *Seq. numbers* field.
+
+  > [Warning] Be aware that the existing bookings will be deleted permanently.
+
+  ![Multiple bookings selected](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/DeleteMultipleBookings02.png "[Multiple bookings selected]")
+
+4. Click the [DELETE] button.   
+The selected bookings are deleted and will no longer be displayed in the bookings list.
+
+> [Info] It may be necessary to click on a different tab and switch back the *BOOKINGS* tab to display the changes in the bookings list.
```

Comment:
Also das ist definitiv ein Bug. Müssen wir reporten. Kann ja nicht sein, dass man da erst so komisch rumwurschteln muss... 
Alternativ müssen wir eben sagen, dass man neu laden muss - hört sich besser an, als den Tab zu wechseln...

Reply:
S. mein Kommentar oben. :) Zwischenlösung: Schritt mit F5 zur Aktualisierung des Platforms, wie in Integration.